app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -15,7 +14,35 @@ const Notes = () => {
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [selectedNoteIds, setSelectedNoteIds] = useState([]);
-  const [notes, setNotes] = useState([]);
+  const [notes, setNotes] = useState([
+    {
+      id: 1,
+      title: "Change support email",
+      description: "forward all internal mails...",
+      tags: { text: "Internal", color: "blue" },
+      createdDate: new Date(),
+      dueDate: new Date(),
+      contact: "Karthik Menon",
+    },
+    {
+      id: 2,
+      title: "Feedback",
+      description: "Feedback V1.0......",
+      tags: { text: "Agile Workflow", color: "green" },
+      createdDate: new Date(),
+      dueDate: new Date(),
+      contact: "Karthik Menon",
+    },
+    {
+      id: 3,
+      title: "feedback Hover",
+      description: "Feedback V2.0......",
+      tags: { text: "Bug", color: "red" },
+      createdDate: new Date(),
+      dueDate: new Date(),
+      contact: "Amal Dinesh",
+    },
```

Comment:
Please move this to a separate constants file. The file need not have `.jsx` extension.